tsconfig.json

```diff
@@ -0,0 +1,42 @@
+{
+  "extends": "@tsconfig/ember/tsconfig.json",
+  "compilerOptions": {
+    "experimentalDecorators": true,
+    "target": "es2017",
```

Comment:
We can probably target something newer than `es2017`, as that's well out of our support matrix.

Reply:
es2020 work?